### PR TITLE
Add urban command

### PR DESCRIPTION
**Pull request by wasdennnoch**
**Please describe the changes this PR makes and why it should be merged:**

**What does this PR do:**
- [x] This PR changes internal functions, modules and/or utilities
  - [ ] This PR modifies the Extension API
- [x] This PR modifies commands
  - [ ] This PR changes command functions
  - [ ] This PR changes metadata for the command, updated in `commands.js`
  - [x] This PR removes and/or adds commands
- [ ] This PR modifies Web processing and/or content
  - [ ] This PR modifies static/ejs content
  - [ ] This PR modifies request processing (endpoint functions)
  - [ ] This PR modifies paths to existing resources
- [ ] This PR **only** includes non-code changes, like changes to default configurations, README, typos, etc.

### CODE REVIEW

File: Commands/Public/urban.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+			]);
+		});
+		const menu = new PaginatedEmbed(msg, descriptions, {
+			color: Colors.RESPONSE,
+			title: `${!msg.suffix ? "Random d" : "D"}efinition {current description} out of {total descriptions}${msg.suffix ? ` for '${msg.suffix}'` : ""}:`,
+			footer: body.tags ? `Tags: ${body.tags.join(", ")}` : "",
+		}, [], fields);
+		await menu.init();
+	} else {
+		winston.verbose(`Failed to fetch Urban Dictionary results`, { svrid: msg.guild.id, chid: msg.channel.id, usrid: msg.author.id, status, err: statusText });
```
Comment (Gilbert142) [inline Commands/Public/urban.js]: Should be winston.debug; winston.verbose is for internal functioning and events, debug is for messages that can aid in debugging issues.

**Review by Gilbert142 — CHANGES_REQUESTED** [review]
Fix this one problem and it should be an approval from me.

File: Commands/Public/urban.js (hunk 1 above)
Comment (wasdennnoch) [reply Commands/Public/urban.js]: Done.

**Review by Gilbert142 — APPROVED** [review]

**Comment by vladfrangu** [comment]
One teeny thing: add 2 \\ before the thumbs up / down emojis. Otherwise LGTM

**Pull request merged**